docs/docs/DefaultApi.md

```diff
@@ -0,0 +1,134 @@
+# AmbianicServerlessFunctionsApiDefinition.DefaultApi
+
+All URIs are relative to *https://ambianic-functions.netlify.app*
+
+Method | HTTP request | Description
+------------- | ------------- | -------------
+[**netlifyFunctionsCustomersGet**](DefaultApi.md#netlifyFunctionsCustomersGet) | **GET** /.netlify/functions/customers | 
+[**subscriberPost**](DefaultApi.md#subscriberPost) | **POST** /subscriber | 
+[**subscriptionDelete**](DefaultApi.md#subscriptionDelete) | **DELETE** /subscription | 
+
+
+
+## netlifyFunctionsCustomersGet
+
+> netlifyFunctionsCustomersGet()
+
+
+
+Serverless function to fetch all customers from a stripe service account.
+
+### Example
+
+```javascript
+import AmbianicServerlessFunctionsApiDefinition from 'ambianic_serverless_functions_api_definition';
+
+let apiInstance = new AmbianicServerlessFunctionsApiDefinition.DefaultApi();
+apiInstance.netlifyFunctionsCustomersGet((error, data, response) => {
+  if (error) {
+    console.error(error);
+  } else {
+    console.log('API called successfully.');
+  }
+});
+```
+
+### Parameters
+
+This endpoint does not need any parameter.
```

Comment:
@vickywane the docs are still a mess. How does `netlifyFunctionsCustomersGet` need no parameter? 
The function name is also poorly chosen. I've already commented on that.

Reply:
I am making changes to improve this in the next push.

Reply:
The `netlifyFunctionsCustomersGet` endpoint needs no parameter. It is responsible for the endpoint that fetches a list of customers within the stripe account. 

From the stripe docs, only a `limit` field is needed to paginate the data response. We currently have no need for pagination. 

See stripe api reference [here](https://stripe.com/docs/api/customers/list)